auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/DeltaUserDetailsRequest.kt

```diff
@@ -120,4 +120,6 @@ data class DeltaUserDetailsRequest(
     @SerialName("organisations") val organisations: List<String>,
     @SerialName("comment") val comment: String? = null,
     @SerialName("classificationType") val classificationType: String? = null, //Not used anywhere yet
+    @SerialName("userObjectGuid") val userObjectGuid: String? = null,
+    @SerialName("isAdmin") val isAdmin: String? = null,
```

Comment:
Why was this change necessary when your Delta PR doesn't seem to change anything about sending these fields?

Reply:
Forgot to remove this from when I tried to send the userDetails field from Delta along with the comment